pages/content/amp-dev/documentation/guides-and-tutorials/contribute/integrate-your-tech.md

```diff
@@ -13,7 +13,7 @@ If you are a technology provider for publishers or advertisers on the web, we in
 There are 4 primary ways you can integrate with AMP:
 
 ## 1. Add your support to the `amp-analytics` extension
-AMP analytics allows you to send events back to your server based on triggers configured by you. We have written an [analytics integration guide]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/configure-analytics/index.md', locale=doc.locale).url.path}}) to get you started.
+AMP analytics allows you to send events back to your server based on triggers configured by you. We have written an [analytics integration guide]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/configure-analytics/configure_analytics.md', locale=doc.locale).url.path}}) to get you started.
```

Comment:
I'm not sure this is better. Index.html was the right choice here, but the URL should be:

https://amp.dev/documentation/guides-and-tutorials/optimize-measure/configure-analytics/

and not

https://amp.dev/documentation/guides-and-tutorials/optimize-measure/configure-analytics/index.html

Reply:
I'm a bit torn: see `.../start/add_advanced/index.html` for example which has been `.../start/add_advanced.html` before. [Add advanced](https://amp.dev/documentation/guides-and-tutorials/start/add_advanced/add_advanced.html?format=websites) is actually a step-by-step tutorial. Here it makes sense to have the `index.html` as its like the entry point of the folder.

`.../optimize-measure/configure-analytics/` on the other hand is a loose collection of documents where `configure_analytics.md` in my opinion has been wrongly declared as the `index.html`.

Nevertheless the exact scheme you describe is not possible as Grow will run into conflicts during build as it would write `.../optimize-measure/configure-analytics` as a file and then fail on `.../configure-analytics/analytics_basics.html` with `Path is not a directory`. That's what I tried to explain earlier.

Reply:
Afaik, `.../optimize-measure/configure-analytics/` is the top-level doc and the others are children.

Reply:
Changed it to `/optimize-measure/configure-analytics/` in the documents frontmatter.